src/components/EmailAddress.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  export let emails;
+</script>
+
+<style>
+  td {
+    @apply border;
+    @apply p-2;
+  }
+</style>
+
+<tr>
+  <td>Notification Email{emails.length > 1 ? 's' : ''}</td>
+  <td>
```

Comment:
I think we should not include the table elements here-- we should focus the component purely on displaying email addresses (we may want to use different elements to organize stuff in different places, but still use this component).